examples/HastyNoise/config.cmake

```diff
@@ -0,0 +1,4 @@
+hunter_config(Boost VERSION ${HUNTER_Boost_VERSION}
+    CMAKE_ARGS
+        BUILD_SHARED_LIBS=ON 
```

Comment:
Probably you should mention this peculiarity in the documentation, e.g.:
- https://docs.hunter.sh/en/latest/packages/pkg/tvm.html